lib/definitions/global.d.ts

```diff
@@ -5,5 +5,5 @@ declare type Environment = "development" | "production" | "test";
 declare type Optional<T> = T | undefined;
 
 declare interface Window {
-    __REDUX_DEVTOOLS_EXTENSION__?: () => string;
+    devToolsExtension: () => () => void;
```

Comment:
Not sure if this is proper, it complained when I just had () => Function and told me to avoid using the Function type, prefer specifying what kind of function. But in this case, I don't know what devToolsExtension exactly returns, I just know it returns a function which also returns a function that might not be void (think it's actually another function), but how many levels deep do I need to go?